### PR TITLE
Add Learning Resource Directory

### PR DESCRIPTION
**Pull request by IritaSee**
Since we have so many interest and reading is gateway to the world as the old says, I suggest that we have a directory contains links, books, articles, publications, etc. So in the future if we want to lookup something important to our profession, we can always rely our small and humble well-designed library.

### CODE REVIEW

**Comment by fakhrip** [comment]
Hmm yeah thats right, this is a good idea indeed !

But i would suggest you to also create a small but well-explained description for each of those resources, so that people wont need to open the books and stuff to understand the context of what they would want to read about @IritaSee

**Comment by IritaSee** [comment]
> Hmm yeah thats right, this is a good idea indeed !
> 
> But i would suggest you to also create a small but well-explained description for each of those resources, so that people wont need to open the books and stuff to understand the context of what they would want to read about @IritaSee

Whoa, nice idea... sure i will add short desc in a readme in each category, thanks for the advice!

**Comment by IritaSee** [comment]
Glossary added! Happy reading!

File: README.md [hunk 1]
```diff
@@ -6,18 +6,21 @@ Selamat datang kalian semua pada pejuang coding yang ingin berjuang untuk mengik
...
   > Sebagai mentor
-- [Muhammad Fakhri Putra Supriyadi](https://github.com/fakhrip) [FAI]
+- [Muhammad Fakhri Putra Supriyadi](https://github.com/fakhrip) [FAI 2017]
   > Sebagai mentor
-- [Dini Rizqi Amalia](https://github.com/dinudinni) [DIN]
+- [Dini Rizqi Amalia](https://github.com/dinudinni) [DIN 2020] 
   > Sebagai mentee
-- [Soediponegoro](https://github.com/Soedipo) [DIP]
+- [Soediponegoro](https://github.com/Soedipo) [DIP 2020]
   > Sebagai mentee
```
Comment (fakhrip) [inline README.md]: I think its better to just leave the formatting as it was, because other mentee would need to adapt to this changes if this one has to change this way

File: resources/AI_and_data_science/DL/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Glosarium
+Berfungsi agar bila kita ingin mencari suatu topik khusus, gak perlu buka semua resourcenya satu per satu. Berisi deskripsi singkat mengenai resource yang ada di folder ini. Bila ingin menkontribusi bacaan jangan lupa di buat ringkasannya dan di tambahkan di sini.
+
+1) How to make the most out of BERT fine tuning
+```
+BERT adalah tokenizer/language processor yang cukup populer dikalangan machine learning engineer. Selain mempermudah hidup dengan gak buat tokenizer sendiri, BERT juga di kostumisasi sesuai dengan bahasa yang kita hadapi. Namun seringkali walaupun udah pake BERT, hasilnya masih gak se impressive yang kita inginkan. Sementara kalau mau fine tuning sendiri, BERT ini agak sesnsitif, salah sedikit malah makin jelek. That's why this PDF is exist. Fine tuning yang di purpose adalah dengan memberikan early stop dan/atau mengatur setiap epoch harus diapain
+```
```
Comment (fakhrip) [inline resources/AI_and_data_science/DL/README.md]: Description like this will be better not to use a code-wrapper "\`\`\`" because it would then becomes a one line full of text and people would have to scroll through a long one liner stuff ....

DeepinScreenshot_select-area_20210805092712

File: resources/AI_and_data_science/ML/README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+2) Why is my validation loss is lower than my training loss
+```
+seringkali dalam kasus pembuatan model ML/DL kita tranining lalu di ukur hasil training per iterasi dengan data validasi. Logikanya validation loss lebih tinggi dari training loss karena data validasi dia baru lihat pertama kali jadi kemungkinan salahnya lebih tinggi... tapi ada lho kasus sebaliknya. Ini merupakan gejala kalau model kita perlu di tune ulang untuk menghindari underfitting, osilasi, dan bahkan overfitting di iterasi-iterasi selanjutnya. Nah PDF ini membahas mengapa hal itu bisa terjadi dan apa yang harus dilakukan.
+```
+
+3) An overview of gradient descent (based) optimization
+```
+Kita mungkin tau berbagai fungsi optimisasi yang ada seperti Stochastic Gradient Descend (SGD), atau variasinya seperti Adam dan Momentum. Tapi apa sih sebenarnya itu dan bagaimana perumusannya? Di PDF ini kita akan mendapatkan semuanya termasuk rumus yang di jalankan di tiap iterasi sehingga membuat model lebih baik lagi bercermin dari kesalahan yang di tampilkan loss function. (Peringatan: intensive math inside)
+```
```
Comment (fakhrip) [inline resources/AI_and_data_science/ML/README.md]: Change this one too

File: resources/AI_and_data_science/data_science/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+berisi nyaris semua rumus, paradigma, tahapan, 'jurus' dan permodelan bila ingin menyelesaikan suatu masalah dengan pendekatan sains data dan kecerdasan buatan. (Peringatan: math inside).
+```
+2) (**Critical**) Beda Korelasi dan Kausalitas
+```
+ini adalah yang membedakan logika orang biasa dan logika kita yang ingin mendalami sains data.  ** Korelasi tidak menyebabkan kausalitas **. Wajib dibaca buat yang ingin mendalami learning path sains data dan kecerdasan buatan. Karena minimal kita harus bisa membedakan korelasi dan kausalitas.
+```
+3) Data Analysis Report Example
+```
+Contoh format bagaimana menyajikan hasil eksplorasi data sebagai ilmuan data (data scientist) ke masyarakat/client/stakeholder dalam dokumen formal. Contoh kasus yang digunakan adalah `Analisis efektifitas PSBB terhadap kasus positif harian COVID-19 di DKI Jakarta`.
+```
```
Comment (fakhrip) [inline resources/AI_and_data_science/data_science/README.md]: Also this one

**Review by fakhrip — CHANGES_REQUESTED** [review]
Please do the changes as needed ...
and thanks for the PR anyway

File: README.md (hunk 1 above)
Comment (fakhrip) [reply README.md]: And for the possibility of having double assistant code could actually be solved by having their full name too, thats also the reason why i included the full name in the formatting

**Comment by IritaSee** [comment]
Formatting fixed, ready for another review

File: resources/AI_and_data_science/ML/README.md [hunk 5]
```diff
@@ -2,16 +2,14 @@
...
-```
+\
 Banyak di antara kita yang sedang belajar machine learning mendapatkan miskonsepsi tentang learning rate. Yang kita tau cuman semakin kecil semakin presisi belajarnya, which is training semakin lama, cuman hasil (harusnya/biasanya) semakin bagus. Ya berarti logikanya kita mau pake learning rate terkecil dong? Gak gitu.. Maka dari itu pengertian matematis dari learning rate juga diperlukan, dan ada di PDF ini. (Peringatan: math inside)
-```
+
 
 2) Why is my validation loss is lower than my training loss
-```
+\
 seringkali dalam kasus pembuatan model ML/DL kita tranining lalu di ukur hasil training per iterasi dengan data validasi. Logikanya validation loss lebih tinggi dari training loss karena data validasi dia baru lihat pertama kali jadi kemungkinan salahnya lebih tinggi... tapi ada lho kasus sebaliknya. Ini merupakan gejala kalau model kita perlu di tune ulang untuk menghindari underfitting, osilasi, dan bahkan overfitting di iterasi-iterasi selanjutnya. Nah PDF ini membahas mengapa hal itu bisa terjadi dan apa yang harus dilakukan.
```
Comment (fakhrip) [inline resources/AI_and_data_science/ML/README.md]:
```suggestion
Seringkali dalam kasus pembuatan model ML/DL kita tranining lalu di ukur hasil training per iterasi dengan data validasi. Logikanya validation loss lebih tinggi dari training loss karena data validasi dia baru lihat pertama kali jadi kemungkinan salahnya lebih tinggi... tapi ada lho kasus sebaliknya. Ini merupakan gejala kalau model kita perlu di tune ulang untuk menghindari underfitting, osilasi, dan bahkan overfitting di iterasi-iterasi selanjutnya. Nah PDF ini membahas mengapa hal itu bisa terjadi dan apa yang harus dilakukan.
```
Use a capital for all starting letter in a sentence

File: resources/AI_and_data_science/ML/README.md [hunk 6]
```diff
@@ -2,16 +2,14 @@
...
-```
 
-3) An overview of gradient descent (based) optimization
-```
+
+3) An overview of gradient descent (based) optimization \
 Kita mungkin tau berbagai fungsi optimisasi yang ada seperti Stochastic Gradient Descend (SGD), atau variasinya seperti Adam dan Momentum. Tapi apa sih sebenarnya itu dan bagaimana perumusannya? Di PDF ini kita akan mendapatkan semuanya termasuk rumus yang di jalankan di tiap iterasi sehingga membuat model lebih baik lagi bercermin dari kesalahan yang di tampilkan loss function. (Peringatan: intensive math inside)
```
Comment (fakhrip) [inline resources/AI_and_data_science/ML/README.md]: If you want to use the backslash on the bottom part of the title, then use it in all of them to uniform it

File: resources/AI_and_data_science/data_science/README.md [hunk 7]
```diff
@@ -2,15 +2,15 @@
 Berfungsi agar bila kita ingin mencari suatu topik khusus, gak perlu buka semua resourcenya satu per satu. Berisi deskripsi singkat mengenai resource yang ada di folder ini. Bila ingin menkontribusi bacaan jangan lupa di buat ringkasannya dan di tambahkan di sini.
 
 1) Data Science Cheatsheet
-```
+\
 berisi nyaris semua rumus, paradigma, tahapan, 'jurus' dan permodelan bila ingin menyelesaikan suatu masalah dengan pendekatan sains data dan kecerdasan buatan. (Peringatan: math inside).
```
Comment (fakhrip) [inline resources/AI_and_data_science/data_science/README.md]: Use capital

File: resources/AI_and_data_science/data_science/README.md [hunk 8]
```diff
@@ -2,15 +2,15 @@
...
-```
+
 2) (**Critical**) Beda Korelasi dan Kausalitas
-```
-ini adalah yang membedakan logika orang biasa dan logika kita yang ingin mendalami sains data.  ** Korelasi tidak menyebabkan kausalitas **. Wajib dibaca buat yang ingin mendalami learning path sains data dan kecerdasan buatan. Karena minimal kita harus bisa membedakan korelasi dan kausalitas.
-```
+\
+ini adalah yang membedakan logika orang biasa dan logika kita yang ingin mendalami sains data.  **Korelasi tidak menyebabkan kausalitas**. Wajib dibaca buat yang ingin mendalami learning path sains data dan kecerdasan buatan. Karena minimal kita harus bisa membedakan korelasi dan kausalitas.
```
Comment (fakhrip) [inline resources/AI_and_data_science/data_science/README.md]: Use capital

**Review by fakhrip — CHANGES_REQUESTED** [review]

**Comment by IritaSee** [comment]
Capitalisations are fixed, branch synchronised with main, ready for merge

**Review by fakhrip — APPROVED** [review]
Nicee you can merge it now
